mimic/test/test_maas.py

```diff
@@ -145,11 +145,27 @@ def test_list_entity(self):
         """
         test list entity
         """
+        marker = ""
```

Comment:
Any reason the `marker` is assigned an empty string before `marker = data['metadata']['next_marker']`